testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/OIDCProtocolMappersTest.java

```diff
@@ -594,6 +594,51 @@ public void testRolesAndAllowedOriginsRemovedFromAccessToken() throws Exception
         }
     }
 
+    /**
+     * Chris Nurse 20-Feb-2022
+     * 
+     * PR-4205 serialised single values as an array. This is not desired when the Multivalued options is set to false.
+     * 
+     * This test ensures singular values are not wrapped as an array.
+     * 
+     * The subsequent test for PR-4205 will continue to test functionality where multivalued = true
+     * 
+     * @throws Exception
+     */
+    @Test
+    public void testUserRoleToAttributeMappersWithMultiValuedRolesEqualsFalse() throws Exception {
```

Comment:
Please consider a new name of the test that doesn't use a negation (for example "...WithSingleValuedRoles")